src/components/collapsible/Collapsible.js

```diff
@@ -50,11 +58,26 @@ class Collapsible extends React.Component<Props, State> {
 
     constructor(props: Props) {
         super(props);
+        this.transitionElement = React.createRef();
         this.state = {
             isOpen: props.isOpen,
+            height: '0',
         };
     }
 
+    componentDidMount() {
+        this.updateTransitionElementHeight();
+    }
+
+    updateTransitionElementHeight = () => {
+        const { current } = this.transitionElement;
+        if (current) {
+            this.setState(prevState => ({
```

Comment:
Can this property be set at the same time as the `this.setState` call in `toggleVisibility`?